docker-compose.yml

```diff
@@ -89,6 +89,17 @@ services:
     - "127.0.0.1:6391:6379"
 
 # Dependencies
+  cosmosdb-emulator:
+    image: mcr.microsoft.com/cosmosdb/linux/azure-cosmos-emulator:latest
+    container_name: cosmosdb-emulator
+    environment:
+      - AZURE_COSMOS_EMULATOR_PARTITION_COUNT=2
+      - AZURE_COSMOS_EMULATOR_ENABLE_DATA_PERSISTENCE=false
+      - AZURE_COSMOS_EMULATOR_IP_ADDRESS_OVERRIDE=127.0.0.1
+    ports:
+      - "8081:8081"
```

Comment:
8081 is a _kind_ of risky choice, as it's a commonly used port. We _might_ want to publicly expose a different port and map it to the internal 8081, but we can leave it for now 👍